pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -208,6 +215,35 @@ func filterSeedsMatchingProviders(cloudProfile *gardencorev1beta1.CloudProfile,
 	return matchingSeeds, nil
 }
 
+// filterSeedsMatchingMultiZonalAttribute filters seeds by label `seed.gardener.cloud/multi-zonal`,
+// based on whether the shoot requires a multi-zonal control plane or not
+func filterSeedsMatchingMultiZonalAttribute(seedList []gardencorev1beta1.Seed, shoot *gardencorev1beta1.Shoot) ([]gardencorev1beta1.Seed, error) {
```

Comment:
`docs/concepts/scheduler.md` describes the scheduling strategy. This changehas to be reflected in this document.